content/zh/docs/setup/upgrade/helm/revlabels_test.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
```

Comment:
should not to sync this test script file, CI will not run test in `content/en`

Reply:
Thanks. This PR is a little big. I'm struggling in the errors -_! Thanks.

Reply:
Let me try to divide this PR to a smaller one before your review.

Reply:
Done! Removal in a PR and added localization in the other PR now. 😄